src/rules/de.toml

```
@@ -7,12 +7,13 @@ quote_start_with_letter = true
 needs_punctuation_end = false
 needs_letter_start = true
 needs_uppercase_start = true
-allowed_symbols_regex = "[\u0020-\u007BäöüßÄÖÜ„“‚‘’–\\.]"
+allowed_symbols_regex = "[\u0020-\u007BäöüßÄÖÜ„“‚‘’–\\.?!]"
 disallowed_symbols = []
 broken_whitespace = ["  ", " ,", " .", " ?", " !", " ;"]
 
 matching_symbols = [
-  ["„", "“"]
+  ["„"«, "“"»],
```

```suggestion
  ["„", "“"],
```
This fixes the syntax error. And the other type of quotes is generally not allowed anyway.